specification/billing/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the Billing API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2020-05
+tag: package-2021-10
```

Comment:
Why is this changing?

Reply:
My understanding is that the readme should point to the most recent api version tag. I assumed that the readme was not in sync with the changes. Thoughts? @wilcobmsft